### DISCUSSION TITLE
[Android] UI offset border

### DISCUSSION THREAD

**Discussion by Akito22** [ppy/osu#15506] [Q&A] [answered]
Basically notch problem, Xiaomi Mi 10T (or phone with circle notch on top left, other example would be Samsung Galaxy A8s) some UI is getting blocked by it. Is there any way to try to not getting it blocked? I searched in setting for any 'safe-region' or UI offset but there's none, It's kinda annoying and infuriating but a minor problem
![Untitled12_20211107092239](https://user-images.githubusercontent.com/93849523/140629960-0019e417-e6b7-43aa-b92e-64ce8da1df77.png)

**Comment by Joehuu** [comment]
https://github.com/ppy/osu/issues/4330

**Comment by peppy** [comment, marked as answer]
You can adjust the game screen area using the "Screen Scaling" setting. Set it to "everything" and adjust the game into the screen area that doesn't have a notch on it.

**Reply by Akito22** [reply]
Ah, I see! Thanks for the answer. I didn't realize the screen scaling have different option to change the type of scaling it used.
